test/onnx/test_utility_funs.py

```diff
@@ -684,6 +684,46 @@ def forward(self, input, other):
         iter = graph.nodes()
         assert next(iter).kind() == "custom_namespace::custom_op"
 
+    def test_custom_opsets_gelu(self):
+        def gelu(g, self):
+            return g.op("com.microsoft::Gelu", self).setType(self.type())
+
+        from torch.onnx import register_custom_op_symbolic
```

Comment:
This has been used in more than 1 test, so let's move it to top instead of calling it in some tests.

Reply:
Sure